script/Arcueid/Day 3/03_00_ARC03_5B.txt

```diff
@@ -295,35 +296,35 @@ But my sense of reason―――
 [sha:8b723de876de934a1273f3539c6f572bf6b8b000]{
 -- Page 207, Offset 4510.
 -- 何を間違えた訳でもない。
-         I didn't make any mistakes.
+         I didn't exactly do anything wrong.
 }
 [sha:79c2457c55dd0bbbf5de056fe59d13e6cdef694c]{
 -- Page 207, Offset 4511.
 -- 何を選んだ訳でもない。
-         I didn't choose anything.
+         I didn't exactly choose this.
 }
 [sha:60a1e770a0ef2ca9bc4feead5bae7467683b00bb]{
 -- Page 207, Offset 4512.
 -- 俺はただ、家に帰りたかっただけなのに。
-         I just wanted to go home.
+         I had just wanted to go home.
 }
 [sha:b26f5d5bd33ccffaa13fac287029b30c6f219db0]{
 -- Page 208, Offset 4513.
 -- @g　理性は。
 @g
-My sense of reason is―――
+My sense of reason―――
 }
 [sha:755f4e189597ab423519c17c821136ca158cea52]{
 -- Page 208, Offset 4514.
 -- @g　漠然と、今まで<騙|だま>し騙しこなしてきた綱渡りの綱が、これで切れてしまった事を実感した。
 // "騙し騙し" is more strongly "using every method possible" or "using every trick in the book" but I'm not sure this line is wrong since it makes quite a lot more sense.
 @g
-I vaguely understand now that the tightrope I tricked myself into walking has been cut.
+The indistinct tightrope act I've been desperately trying to walk all this time, has been cut.
```

Comment:
The original flowed terribly between this and the above line. That is essentially the whole impetus of the change.
I don't mind it like you've written it though, and I do think you're right that he intended 漠然と to be a qualifier on 実感した

Reply:
I think that works. Or we just quietly drop 漠然と instead of reassociating it, since that's what kinda stutters the link. Or I guess you could yoink it _all_ the way back to the first one?
> But [I realize that] my sense of reason―――
> [...]
> My sense of reason―――
The tightrope act I'd been tricking myself into maintaining all this time――has snapped.

Reply:
I kinda like that solution, it actually makes it more coherent in entirety.